src/hevm/src/EVM/ABI.hs

```diff
@@ -437,7 +437,7 @@ genAbiValue = \case
    AbiTupleType ts ->
      AbiTuple <$> mapM genAbiValue ts
   where
-    genUInt n = AbiUInt n <$> arbitraryIntegralWithMax n
+    genUInt n = AbiUInt n <$> arbitraryIntegralWithMax (2^n-1)
```

Comment:
this is the fix for the property based tests?

Reply:
yeah, I think I need to rebase this